plugins/database/src/mid_level_routines.cpp

```diff
@@ -1380,10 +1380,15 @@ int checkObjIdByTicket( const char *dataId, const char *accessLevel,
 
     char original_collection_name[MAX_NAME_LEN];
     std::vector<std::string> bindVars;
-    bindVars.push_back( dataId );
+    bindVars.emplace_back(dataId);
+    bindVars.emplace_back(dataId);
```

Comment:
please add a little comment explaining the dataId is intentionally/correctly used as a collId - so it doesn't look like a mistake or require additional scrutiny in the future.

Reply:
I can rename the variable to `object_id` assuming that is what you're getting at.

Reply:
no, a comment is plenty fine, no need to change any names/logic.

Reply:
Updated. Please check the comment to make sure it is acceptable.

Reply:
sold.  squash and # it.

Reply:
Done.